app/Http/Controllers/UserController.php

```diff
@@ -4,15 +4,18 @@
 
 use App\Models\User;
 use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Support\Facades\Hash;
 use App\Contracts\Services\UserServiceInterface;
-use App\Services\UserService;
 
 class UserController extends Controller
 {
     /**
      * user interface
      */
     private $userService;
+    const ADMIN_ROLE = 1;
+    const USER_ROLE = 2;
```

Comment:
Declare constant in model. Cause it's model constant. 
Else declare in config/constants.php.
Use class constant only for variable that is useful only for that class.